build/docker/Dockerfile-bare

```diff
@@ -0,0 +1,43 @@
+# This dockerfile builds the zap stable release
+FROM ubuntu:16.04 as builder
```

Comment:
I am using the new [multi stage build](https://docs.docker.com/engine/userguide/eng-image/multistage-build/) feature. So, this is only the build image - to download Zap and extract it. This container is removed after this part, and the real one (See from line 20) is based on alpine.